themes/bootstrap3/templates/myresearch/odmycontent.phtml

```diff
@@ -55,6 +68,8 @@
             </a>
           </h4>
 
+          <div class="od_edition"><?=$this->escapeHtml($edition)?></div>
```

Comment:
Do we want to display this div even if $edition is an empty string? Would it make sense to wrap in `<?php if ($edition): ?> ... <?php endif; ?>`?